ci-operator/jobs/openshift/release/openshift-release-presubmits.yaml

```diff
@@ -28,7 +28,8 @@ presubmits:
         - --job-config-path
         - ./ci-operator/jobs/
         - --plugin-config
-        - ./cluster/ci/config/prow/plugins.yaml
+        - ./cluster/ci/config/prow/plugins.
```

Comment:
is the `yaml` gone at the end of the line intentional?

Reply:
eek